client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -104,6 +105,22 @@ const OptionList = styled.ul`
 	}
 `;
 
+const CurrentOptionContainer = styled.div`
+	align-items: center;
+	display: flex;
+	font-size: 14px;
```

Comment:
Looks like you can use `font-size: ${ ( props ) => props.theme.fontSize.small };` here, I found the values in this file: fbhepr%2Skers%2Spnylcfb%2Scnpxntrf%2Spbzcbfvgr%2Qpurpxbhg%2Sfep%2Syvo%2Sgurzr.gf%3Se%3Q4o0qo534%23110-og

Additionally below you can use `font-weight: ${ ( props ) => props.theme.weights.normal };`